### PR TITLE
Add kibana import job

### PR DESCRIPTION
**Pull request by gnanirahulnutakki**

### CODE REVIEW

**Comment by pgodey** [comment]
Rename config-map.yaml --> kibana-config-map.yaml and helm-job.yaml --> kibana-job.yaml

File: charts/fid/templates/helm-job.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    spec:
+      restartPolicy: Never
+      volumes:
+      - name: kibana-data
+        configMap:
+          name: kibana-data
+          defaultMode: 0777
+      containers:
+      - name: kibana-import
+        image: centos:latest
```
Comment (pgodey) [inline charts/fid/templates/helm-job.yaml:21]: Why are you using centos image? Can you use busybox?

**Review by pgodey — CHANGES_REQUESTED** [review]
Rename config-map.yaml to kibana-config-map.yaml and helm-job.yaml to kibana-job.aml

File: charts/fid/templates/helm-job.yaml (hunk 1 above)
Comment (gnanirahulnutakki) [reply charts/fid/templates/helm-job.yaml:21]: I did not know if busybox had curl, let me try that and check if that works too

File: charts/fid/templates/helm-job.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: KIBANA_URL
```
Comment (pgodey) [inline charts/fid/templates/helm-job.yaml]: I support this is kibana host, right? I would rather have one parameter called KIBANA_URL that has value like http://<host>:<port> and use that in the script.

**Review by pgodey — CHANGES_REQUESTED** [review]

**Pull request merged**